experiment_helpers/randomize_stimuli.m

```diff
@@ -1,12 +1,15 @@
-function [expt] = randomize_stimuli(expt, nTrialsPerPert, nTrialsPerNonpert, nBaseline, nWashout)
-% Pseudorandomize words and conditions for experiments.
+function [expt] = randomize_stimuli(expt, nTrialsPerPert, nTrialsPerNonpert, nBaseline, nWashout, bAlternatePnp)
+% Pseudorandomize words and conditions for experiments. Specifically meant for experiments that have more than one
+% perturbation AND more than one word. For experiments where you are only randomizing one of the variables, see
```

Comment:
I'd add "for experiment where you are only randomizing one of the variables (for example, most adaptation experiments only randomize expt.words), see ..."

I considered recently just changing the name of this function to "randomize_stimuli_comp" because it's really only for compensation experiments. Now it's a bit more general, but I like having a pointer/reminder that for most adaptation stuff you don't need this function.